### PR TITLE
Add service events

### PR DESCRIPTION
**Pull request by riverzhang**
I think I should know what happened to the service resource.

setup 1 example  my-service.yaml 
kind: Service
apiVersion: v1
metadata:
  name: my-service
spec:
  selector:
    app: MyApp
  ports:
    - protocol: TCP
      port: 80
      targetPort: 9376
      nodePort: 30061
  clusterIP: 10.254.63.159
  loadBalancerIP: 10.254.63.158
  type: LoadBalancer
status:
  loadBalancer:
    ingress:
      - ip: 10.254.63.190
setup 2   kubectl   create -f  my-service.yaml
setup 3   kubectl describe service my-service
Name: my-service
Namespace: default
Labels: <none>
Annotations: <none>
Selector: app=MyApp
Type: LoadBalancer
IP: 10.254.63.159
Port: <unset> 80/TCP
NodePort: <unset> 30061/TCP
Endpoints: <none>
Session Affinity: None
Events:
FirstSeen LastSeen Count From SubObjectPath Type Reason Message
--------- -------- ----- ---- ------------- -------- ------ -------
15s 9s 2 service-controller Normal CreatingLoadBalancer Creating load balancer
14s 7s 2 service-controller Warning CreatingLoadBalancerFailed Error creating load balancer (will retry): Failed to create load balancer for service default/my-service: Error creating loadbalancer a023c8df05c0111e7ae13fa163ecc72d: Error creating loadbalancer
{a023c8df05c0111e7ae13fa163ecc72d Kubernetes external service a023c8df05c0111e7ae13fa163ecc72d 11741c92-2a4a-42b4-8dd5-0b35566cf9b2 10.254.63.158 <nil> }

: Invalid request due to incorrect syntax or missing required parameters.

### CODE REVIEW

File: src/app/backend/resource/service/detail.go [hunk 1]
```diff
@@ -53,18 +53,21 @@ type ServiceDetail struct {
...
-func GetServiceDetail(client k8sClient.Interface, heapsterClient heapster.HeapsterClient,
+func GetServiceDetail(client k8sClient.Interface, heapsterClient client.HeapsterClient,
 	namespace, name string, dsQuery *dataselect.DataSelectQuery) (*ServiceDetail, error) {
 
 	log.Printf("Getting details of %s service in %s namespace", name, namespace)
-
+	
 	// TODO(maciaszczykm): Use channels.
-	serviceData, err := client.CoreV1().Services(namespace).Get(name, metaV1.GetOptions{})
+	serviceData, err := client.Core().Services(namespace).Get(name, metaV1.GetOptions{})
```
Comment (floreks) [inline src/app/backend/resource/service/detail.go]: We intentionally use explicit version here as documentation suggests:

```golang
// Deprecated: Core retrieves the default version of CoreClient.
// Please explicitly pick a version.
func (c *Clientset) Core() corev1.CoreV1Interface {
	if c == nil {
		return nil
	}
	return c.CoreV1Client
}
```

Comment (riverzhang) [inline src/app/backend/resource/service/detail.go]: Thanks, I modified it.

File: src/app/backend/resource/service/detail.go [hunk 2]
```diff
@@ -62,7 +65,7 @@ func GetServiceDetail(client k8sClient.Interface, heapsterClient heapster.Heapst
 	namespace, name string, dsQuery *dataselect.DataSelectQuery) (*ServiceDetail, error) {
 
 	log.Printf("Getting details of %s service in %s namespace", name, namespace)
-
+
```
Comment (maciaszczykm) [inline src/app/backend/resource/service/detail.go]: Did you run `gulp format`?

Comment (riverzhang) [inline src/app/backend/resource/service/detail.go]: run  gulp format  result:
gulp format
[17:08:13] Requiring external module babel-register
[17:08:41] Using gulpfile ~/golang/lib/src/github.com/kubernetes/dashboard/gulpfile.babel.js
[17:08:41] Starting 'format-javascript'...
[17:08:41] Starting 'format-html'...
[17:08:41] Starting 'format-go'...
[17:08:43] Finished 'format-go' after 2.37 s
[17:08:46] Finished 'format-html' after 4.58 s
[17:08:50] Finished 'format-javascript' after 9.17 s
[17:08:50] Starting 'format'...
[17:08:50] Finished 'format' after 7.67 μs

Comment (maciaszczykm) [inline src/app/backend/resource/service/detail.go]: It formats source code, you should commit the changes :)

Comment (riverzhang) [inline src/app/backend/resource/service/detail.go]: Thanks, I understand that.

**Comment by riverzhang** [comment]
@maciaszczykm     
codecov/patch — No report found to compare against
codecov/project — No report found to compare against

What's wrong with that?

**Comment by maciaszczykm** [comment]
@riverzhang Don't worry about it. These errors occur occasionally, it is fine.

**Comment by riverzhang** [comment]
@maciaszczykm   I added the front part of the code. Could you please reivew my code.

File: src/app/frontend/service/detail/actionbar.html [hunk 3]
```diff
@@ -14,6 +14,13 @@
 limitations under the License.
 -->
 
+<md-button ng-click="$ctrl.handleScaleResourceDialog()">
+  <md-icon class="kd-actionbar-icon-button">linear_scale</md-icon>
+  <md-tooltip md-direction="bottom">
+    [[Edit number of pods|Tooltip for the 'scale' button on the action bar of a deployment details view.]]
+  </md-tooltip>
+  [[Scale|Tooltip for the 'scale' button on the action bar of a deployment details view.]]
+</md-button>
```
Comment (maciaszczykm) [inline src/app/frontend/service/detail/actionbar.html]: How do you want to scale a service?

image

Comment (riverzhang) [inline src/app/frontend/service/detail/actionbar.html]: Sorry!  K8s does not support service scaling. I delete it.

Comment (riverzhang) [inline src/app/frontend/service/detail/actionbar.html]: @maciaszczykm   I fix it.

**Comment by maciaszczykm** [comment]
Thanks for contribution!

**Pull request merged**